### PR TITLE
Feature/sass migrator

### LINKED ISSUE #16: DEPRECATION WARNING: Using / for division is deprecated and will be removed in Dart Sass 2.0.0.
More information https://sass-lang.com/documentation/breaking-changes/slash-div

### PR DESCRIPTION
**Pull request by KasperAndersson**
Closes https://github.com/slavanga/baseguide/issues/16, or at least draft to..

cc @slavanga @W4ldgeist

### CODE REVIEW

**Comment by slavanga** [comment]
@KasperAndersson Hi Kasper, thank you for taking the time to submit this. I will review it 😊

File: .nvmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+14
```
Comment (slavanga) [inline .nvmrc]: @KasperAndersson Let's use 16.15.1 (latest LTS) here, then regenerate the lock file.

File: package.json [hunk 2]
```diff
@@ -9,6 +9,16 @@
...
+  "scripts": {
+    "dev": "gulp",
+    "build": "gulp build",
+    "watch": "gulp watch",
+    "lint": "gulp lint"
+  },
+	"engines": {
+		"node": "14",
+		"npm": "8"
+	},
```
Comment (slavanga) [inline package.json]: Is this really needed? If yes please update to the LTS versions as well.
Also: should be indented using spaces.

File: package.json [hunk 3]
```diff
@@ -30,13 +40,14 @@
...
     "gulp-rename": "^2.0.0",
-    "gulp-sass": "^4.1.0",
+    "gulp-sass": "^5.1.0",
     "gulp-sourcemaps": "^3.0.0",
     "gulp-uglify": "^3.0.2",
-    "postcss": "^8.2.8",
-    "postcss-scss": "^3.0.5",
+    "postcss": "^8.4.14",
+    "postcss-scss": "^4.0.4",
+		"sass": "^1.53.0",
```
Comment (slavanga) [inline package.json]: Indent using spaces

**Review by slavanga — CHANGES_REQUESTED** [review]
@KasperAndersson looks very good overall. 

I left a few comments. Could you also add a new line after the last `@use` everywhere?

**Review by slavanga — APPROVED** [review]

**Comment by slavanga** [comment]
@KasperAndersson perfect 👌

**Comment by KasperAndersson** [comment]
Hi @slavanga , do you plan to release this to npm registry?

**Comment by slavanga** [comment]
Hey @KasperAndersson the changes are released now:
https://www.npmjs.com/package/baseguide/v/4.2.0

**Pull request merged**